internal/cmd/broker/delete_broker.go

```diff
@@ -94,6 +99,15 @@ func (dbc *DeleteBrokerCmd) HideUsage() bool {
 	return true
 }
 
+// AskForConfirmation asks the user to confirm deletion
+func (dbc *DeleteBrokerCmd) AskForConfirmation() (bool, error) {
+	if !dbc.force {
+		message := fmt.Sprintf("Really delete brokers with names [%s]: ", strings.Join(dbc.names, ", "))
```

Comment:
```go
fmt.Sprintf("Do you really want to delete brokers with names [%s] (Y/n): ", strings.Join(dbc.names, ", "))
```

I think it's a little more clearer to the user. What do you think about this kind of message?